ykcapi/src/lib.rs

```diff
@@ -148,20 +190,38 @@ pub extern "C" fn yk_stopgap(
                     8 => unsafe { ptr::read::<u64>(addr as *mut u64) as u64 },
                     _ => unreachable!(),
                 };
+                let aot = &aotmap[i];
+                unsafe {
+                    sginterp.init_live(
+                        aot.bbidx,
+                        aot.instridx,
+                        std::ffi::CStr::from_ptr(aot.fname),
+                        SGValue::U64(v),
+                    );
+                }
                 // FIXME: remove once we have a stopgap interpreter.
-                eprintln!("Indirect: {} ({} {})", v, reg, off);
+                eprintln!("Indirect: {} ({} {} {})", v, reg, off, size);
```

Comment:
Since we now have a stopgap interpreter, shouldn't this disappear?

Reply:
Yes, once this stops being a draft I'll remove those.